frontend/src/Components/StoreSearch/StoreSearch.js

```diff
@@ -0,0 +1,114 @@
+import React from 'react';
+import StoreCard from "./StoreCard";
+import {Container, Col, Row, Jumbotron, Button, ButtonGroup, Card, InputGroup, FormControl} from "react-bootstrap";
+import './StoreSearch.css'
+
+const stores = [
+    {
+        name: "Super Market",
+        address: "123 E Green St. Champaign, IL",
+        hours: [9, 18],
+        distance: 6
+    },
+    {
+        name: "Local Grocer",
+        address: "456 Testing Ave. Boston, MA",
+        hours: [8, 16],
+        distance: 11
+    }
+    ]
+
+const getLocation = () => {
+    if (navigator.geolocation) {
+        navigator.geolocation.getCurrentPosition(processLocation, locationError);
```

Comment:
Do we want to pass in a PositionOptions object with the timeout set to something also, to be safe? Because it looks like by default the PositionsOptions.timeout is set to Infinity (https://developer.mozilla.org/en-US/docs/Web/API/PositionOptions)

Reply:
Added w/ newest commit.